public/main.js

```diff
@@ -1,29 +1,156 @@
 /* global $ */
 
+let teamData = []
 function fetchTeamData(path) {
   fetch(path)
     .then(res => res.json())
     .then(data => {
-      console.log(data)
+      teamData = data
+      sortExpRating(teamData)
```

Comment:
you can reuse your `sortDataDescending` function here, and pass in the correct attribute, so no need for the `sortExpRating` function